mmv1/api/resource/examples.go

```diff
@@ -179,8 +179,8 @@ func (e *Examples) UnmarshalYAML(n *yaml.Node) error {
 
 // Executes example templates for documentation and tests
 func (e *Examples) SetHCLText() {
-	docCopy := e
-	testCopy := e
+	docVars := make(map[string]string)
+	testVars := e.TestEnvVars
```

Comment:
This line and line 206 are not needed?

Reply:
documentation and tests need to have different `TestEnvVars`. Previously, I had 
```
	docCopy := e
	testCopy := e

Reply:
183 keeps a reference of the original `TestEnvVars`, 203 temporarily switches to the new vars merged with `doc_defaults`, and then 206 switches it back.

Reply:
Got it. Thanks.